benchmark/metamorpheus_Process.R

```diff
@@ -0,0 +1,41 @@
+calculate_Metrics <- function(QuantData, protein_mappings, label){
+
+    # dataProcessPlots(QuantData, "QCPlot", which.Protein = "allonly", address = FALSE, isPlotly = TRUE)
+
+    comparison <- matrix(c(-1,0,0,0,1, # 3x
+                        -1,0,0,1,0, # 2.5x
+                        -1,0,1,0,0, # 2x
+                        -1,1,0,0,0),nrow=4,byrow = TRUE) # 1.5x
+                        
+    row.names(comparison) <- c("E-A", "D-A", "C-A", "B-A")
+    groups = levels(QuantData$ProteinLevelData$GROUP)
+    colnames(comparison) <- groups[order(as.numeric(groups))]
+    model <- groupComparison(contrast.matrix=comparison, data=QuantData,
+                                    use_log_file = FALSE)
+                    
+                                    
+                                    
+    ecoli_proteins = protein_mappings %>% filter(Organism == "Escherichia coli (strain K12)")
+    model$ComparisonResult = model$ComparisonResult %>% mutate(ecoli = Protein %in% ecoli_proteins$`Protein Groups`)
+
+
+    e_group = model$ComparisonResult %>% filter(Label == "B-A") %>% filter(is.na(issue))
```

Comment:
Looks like you hard-coded `B-A` here.  Also, would change the variable name away from `e_group` to maybe like `filtered_results`.